scripts/ninja.js

```diff
@@ -102,6 +102,7 @@ function generateVisitorPattern() {
  *
  */
 var useEnv = false;
+var BS_NATIVE = false
 
```

Comment:
so this file is only for dev build, can you add a flag to toggle the env variable directly, e.g,
```js
function setNative(){
   process.env.BS_NATIVE = true
}
```

Reply:
Done!

Reply:
do you know why?

Reply:
My understanding is that you have 2 rules to build `bsb.exe`, one for dev (which simply calls ocamlopt on all files and then links them) and one for release (which calls bspack and then builds that). Keeping the`BS_NATIVE` inlined in the generated build.ninja helps for the first case